src/middlewared/middlewared/plugins/mail.py

```diff
@@ -283,11 +283,9 @@ def send_raw(self, job, message, config):
         else:
             interval = timedelta(seconds=interval)
 
-        sw_name = self.middleware.call_sync('system.version').split('-', 1)[0]
-
         channel = message.get('channel')
         if not channel:
-            channel = sw_name.lower()
+            channel = 'truenas'
```

Comment:
You PR comment says "define a static string".  Perhaps that should actually be a definition in a file that gets imported and used here, so that we don't have constants sprinkled throughout our code.

Reply:
Agree. Thanks, I've pushed a fix addressing your review.